tests/test_utils.py

```diff
@@ -63,6 +64,15 @@ def test_can_get_config_file_by_environment_variable(self):
         observed = utils.get_global_config_file()
         self.assertEqual(observed, '/tmp/super_file')
 
+    def test_can_get_custom_ca_certificates_variable(self):
+        os.environ[utils.CUSTOM_CA_CERTS_VAR] = '/tmp/ca-certificates.crt'
+        observed = utils.get_custom_ca_certs_file()
+        self.assertEqual(observed, '/tmp/ca-certificates.crt')
+
+    def test_can_default_get_custom_ca_certificates_variable(self):
```

Comment:
This test name is kind weird, don't you think? test_can_get_default_blablabla is better, no?